core/templates/domain/topics_and_skills_dashboard/topics-and-skills-dashboard-backend-api.service.ts

```diff
@@ -93,7 +93,7 @@ export class TopicsAndSkillsDashboardBackendApiService {
         status: filter.status,
         sort: filter.sort,
         keywords: filter.keywords,
-        num_items_to_fetch: itemsPerPage,
+        num_skills_to_fetch: parseInt(itemsPerPage),
```

Comment:
Shouldn't itemsPerPage just be an int from the start?

Reply:
Yes it should be and it is(Added that just for a layer of security, you are right, not needed). Removed ![image](https://user-images.githubusercontent.com/30312043/85357179-5ce73200-b52e-11ea-98ea-f9a7ff7855b9.png)